posts/app.py

```diff
@@ -96,6 +115,20 @@ def generate_timestamp() -> str:
     return "2017-10-06T00:00:00+00:00"
 
 
+def upload_file_to_cloud_and_get_url(file):
```

Comment:
This method name is kinda long. Maybe just upload_file_to_cloud? And the comments explain that the url is returned.

Reply:
Done https://github.com/knative-portability/large-events/pull/20/commits/e0c5ac22af03a910c3995786f917a0ea206258db